src/organisms/cards/RadioCard/Readme.md

```diff
@@ -8,7 +8,7 @@ No Image:
         super(props);
 
         this.state = {
-          radioValue: '',
+          value: '',
```

Comment:
why was this key changed?

Reply:
```
Unused state field: 'radioValue' 
react/no-unused-state
```

the `state` was declared as `radioValue`, but everywhere else in the file it was referred to as `value`